corehq/form_processor/models.py

```diff
@@ -547,13 +548,13 @@ def get_data(self, path):
     def get_xml(self):
         try:
             return self.get_attachment('form.xml')
-        except NotFound:
+        except AttachmentNotFound:
```

Comment:
Seems like `NotFound` could still be raised here (ex. if the blob is missing)? Does it makes sense to catch both?
```suggestion
        except (AttachmentNotFound, NotFound):
```